### ISSUE TITLE
card images disappear when hovering

### ISSUE THREAD

**Issue by TineyeTarball** [papso-kirby/AlCoMa#1]
hover over a card, image stays only for around 1-2 seconds

**Comment by papso-kirby** [comment]
Yeah, because tool tips with images suck in WinForms. Will propably replace it with a button 'View Card' or something.
